src/Couchbase.Lite.Shared/API/Database/DocumentChangedEventArgs.cs

```diff
@@ -37,7 +37,7 @@ public sealed class DocumentChangedEventArgs : EventArgs
         public string DocumentID { get; }
 
         /// <summary>
-        /// The source of the document that changed
+        /// [DEPRECATED] The source of the document that changed
         /// </summary>
         [NotNull]
         public Database Database { get; }
```

Comment:
This time I am pretty sure that Obsolete is missing

Reply:
oh! hmmm, then there are a bunch of new properties need Obsolete attributes? :s

Reply:
Possibly.  Adding deprecated to the comments is not nearly as noticeable as adding the Obsolete attribute so the attribute is the standard to use when deprecating.

Reply:
Just checked and found out most of the newly deprecated properties are already contain the obsolete attributes. I added the missing ones on couple newly deprecated properties.